README.md

```diff
@@ -41,7 +41,9 @@ After install run the following command in the root. (this will install any depe
 ```bash
 $ composer install
 ```
-
+##SCSS to CSS
+whenever you want to change the CSS of Homepage.php you need to do changes in "style.scss" then compile this .scss file into style.css, as we 
+are adding style.css file in php.
```

Comment:
Please document the tools / commands intended to be used for this compiling, Also i just noticed that SCSS file has been added for only one file, any reasons why it has not been added for others?